docs/models/miscellaneous.rst

```diff
@@ -0,0 +1,132 @@
+.. py:currentmodule:: coc
+
+Miscellaneous
+=============
+
+Graphics
+~~~~~~~~
+
+Badge
+^^^^^
+.. autoclass:: Badge()
+    :members:
+    :private-members:
+
+Icon
+^^^^
+.. autoclass:: Icon()
+    :members:
+    :private-members:
+
+Time
+~~~~
+
+Timestamp
+^^^^^^^^^
+.. autoclass:: Timestamp()
+    :members:
+    :private-members:
+
+
+TimeDelta
+^^^^^^^^^
+.. autoclass:: TimeDelta()
+    :members:
+    :private-members:
+
+Location
+~~~~~~~~
+.. autoclass:: Location()
+    :members:
+    :private-members:
+
+Enumerations
+~~~~~~~~~~~~
+
+The library provides some enumerations for certain types of strings, as well as orders for troops, spells and
+achievements that are used internally.
+
+All elixir troops
```

Comment:
I think these should all be capitalised, unless someone has a good reason for why not.. it's just neater IMO and more consistent with everything else
```suggestion
All Elixir Troops
```